notebooks/official/feature_store/vertex_ai_feature_store_feature_view_service_agents.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<p>Can we set the region to the default us-central1? It'll help the test cleanup in the test framework.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2752/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Currently using us-west2 for testing. We can change to us-central1 before submission.</p>